mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/massindexing/impl/PojoMassIndexingNotifier.java

```diff
@@ -130,6 +162,25 @@ void reportIndexingCompleted() {
 			return;
 		}
 
+		// report other failures that went over the limits:
```

Comment:
That's a good idea, but reporting those failures is exactly what we *don't* want :)

I think you meant this?

```suggestion
		// report that some failures went unreported
```